vars/iqeUtils.groovy

```diff
@@ -488,14 +499,16 @@ private def createTestStages(String appName, Map appConfig) {
 
     stage("Results") {
         def pluginsFailed = pluginResults.findAll { it.value == "FAILURE" }
+        def pluginsError = pluginResults.findAll {it.value == "ERROR" }
```

Comment:
nitpicking here ... can we have the whitespace match the others -- e.g. `{ it.value == "ERROR" }`